.buildbot.sh

```diff
@@ -23,3 +23,13 @@ done
 $CC _main.bc -L. -l_a -l_b -l_cc -l_data -o test
 cd ..
 rm -rfv out
+
+LLVM_EXTRACT=$CHERI/bin/llvm-extract make test-global-dependency CC=$CC CXX=$CHERI/bin/clang++ 
+make CC=$CC test-compile 
```

Comment:
Does the `CC=$CC` bit do anything?

Reply:
It does as env vars don't override the defaults in the makefile.